src/policy/immix/defrag.rs

```diff
@@ -25,6 +25,22 @@ pub struct Defrag {
     available_clean_pages_for_defrag: AtomicUsize,
 }
 
+pub struct PlanStatsForDefrag {
```

Comment:
We may simply rename this to `StatsForDefrag` because the information doesn't ultimately come from the plan.  The total pages, for example, comes from the GC trigger, although we call a method of the plan to get it.  Reserved pages and collection reserved pages are aggregated from the spaces and external sources (such as `VMBinding::VMCollection::vm_live_bytes()`.  So I think we don't need to emphasise "Plan" here.

Reply:
Sounds good.